rhasspywake_raven/dtw.py

```diff
@@ -42,33 +42,24 @@ def compute_path(self) -> typing.Optional[typing.List[typing.Tuple[int, int]]]:
         row = m - 1
         col = n - 1
         path = [(row, col)]
-        eps = 1e-14
-
-        while (row > 0) or (col > 0):
-            if (row > 0) and (col > 0):
-                min_cost = min(
-                    self.cost_matrix[row - 1][col],  # insertion
-                    self.cost_matrix[row][col - 1],  # deletion
-                    self.cost_matrix[row - 1][col - 1],  # match
-                )
 
-                if math.isclose(
-                    min_cost, self.cost_matrix[row - 1][col - 1], rel_tol=eps
-                ):
-                    row = row - 1
-                    col = col - 1
```

Comment:
math.isclose is not really required here, since even with floating point it is completely fine to check for equality with itself. In this case we take the smallest of three values and then compare it with those three values again. So it is guaranteed to be equal to one of those three even with floating point math